examples/model.py

```diff
@@ -0,0 +1,83 @@
+import torch
```

Comment:
why put this file here?

Reply:
So move both load_graph.py and model.py to `gs.utils` would be better?